aui.views/src/AUI/View/AForEachUI.h

```diff
@@ -76,4 +76,5 @@ class AForEachUI: public AViewContainer, public AListModelObserver<T>::IListMode
     }
 };
 
-#define AUI_DECLARATIVE_FOR(value, model, layout) _new<AForEachUI<std::decay_t<decltype(model)>::stored_t::stored_t, layout>>(model) - [&](const std::decay_t<decltype(model)>::stored_t::stored_t& value, unsigned index) -> _<AView>
+#define AUI_DECLARATIVE_FOR_EX(value, model, layout, ...) _new<AForEachUI<std::decay_t<decltype(model)>::stored_t::stored_t, layout>>(model) - [__VA_ARGS__](const std::decay_t<decltype(model)>::stored_t::stored_t& value, unsigned index) -> _<AView>
```

Comment:
Generally, to give the possibility to define custom lambda capture group we define `EX`tended macro without square brackets \[\]. This way the user can define custom capture group and additionally pass any `operator()()`-capable object. Refer to `async` and `asyncX` macros for example.

Reply:
Wanted to do this at first, but I wasn't sure on how because the lambda here already have a specific signature. Declaring the macro without the square brackets will ask the user to manually give the correct signature, or am I missing something?

Reply:
Hm, you are right. Let's keep your solution as is then.